FP2-blobs-out.txt

```diff
@@ -0,0 +1,427 @@
+target/product/FP2/system/vendor/lib/libmmipstreamsourcehttp.so
```

Comment:
I dont see your vendor include anywhere in the repo so the 2nd line is just my assumption:

```
TARGET_DEVICE_BLOBS_LIST := true
TARGET_DEVICE_BLOBS_SRC_DIR := vendor/fairphone/FP2/proprietary
TARGET_DEVICE_BLOBS := device/fairphone_devices/FP2/proprietary-files.txt
```

The proprietary files can just be a list of the files in their expected locations:

proprietary-files.txt:
`vendor/lib/libmmipstreamsourcehttp.so`